.github/workflows/main.yaml

```diff
@@ -22,6 +22,12 @@ jobs:
         with:
           node-version: 18
 
+      # Remove package-lock.json and node_modules directory
+      - name: Clean dependencies
+        run: |
+          rm -rf package-lock.json node_modules
+
+      # Reinstall dependencies
       - name : Npm Install
         run: npm install
```

Comment:
What do you think about adding script `npm run reinstall` to `package.json` and run it in this job (and also use manually)?

Reply:
Great idea!